apps/prairielearn/assets/scripts/instructorAiGenerateDraftsClientPreact.tsx

```
@@ -0,0 +1,20 @@
+import { render } from 'preact';
+import { Fragment, h, render } from 'preact';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'render'. Consider to delete it or rename it.

'render' is defined here:


(lint/suspicious/noRedeclare)